### PR TITLE
README.rst: Add waffle.io graph

### PR DESCRIPTION
**Pull request by andreidinu**
Added a graph with closed PRs and closed issues for the last
month.

Fixes https://github.com/coala/coala/issues/4062

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
I think your account is not linked properly with git, as it doesnt appear the commit is made by you but your name is mentioned (your github account isnt linked). you might want to fix that.

File: README.rst [hunk 1]
```diff
@@ -32,3 +32,10 @@ coala supports popular programming languages including **Python**, **C/C++**, **
    :target: http://docs.coala.io/
 .. |AGPL| image:: https://img.shields.io/pypi/l/coala.svg
    :target: https://www.gnu.org/licenses/agpl-3.0.html
+
+
+.. image:: https://graphs.waffle.io/coala/coala/throughput.svg
+ :target: https://waffle.io/coala/coala/metrics/throughput
+ :alt: 'Throughput Graph'
+
+*Last updated: 27th April, 2017*
```
Comment (sims1253) [inline README.rst]: do we have to manually update the readme?  Thought the image would just change over time.
Or is there another reason for this line?

Comment (andreidinu) [inline README.rst]: I initially thought we have to update it manually. Right now I tried it on one of my repos and the image updates by itself.

**Comment by sims1253** [comment]
On a final note, I am not sure if we want this graph on our page. It is cool and all and we'd be part of the hip agile crowd but It doesn't really serve a purpose I feel.

**Comment by supergr35** [comment]
@sims1253 I'd argue it's something which would be nice for people to see, as it'd tell them our actual level of activity and that we're putting work into our PRs.

**Comment by SanketDG** [comment]
I'd like to get @sils response here, he wanted a minimal readme. Does this help at all?

**Comment by sils** [comment]
I think it's nice, people see that we're active :)

**Comment by supergr35** [comment]
Commit message: `Added` -> `Add`

**Comment by jayvdb** [comment]
@andreidinu , please try rebasing. That should fix the codecov problem.

**Comment by adtac** [comment]
@cherry-pick-bot

**Comment by Adrianzatreanu** [comment]
@adtac so do we merge here or on your cherry pick PR?

**Comment by meetmangukiya** [comment]
@cherry-pick-bot

**Comment by meetmangukiya** [comment]
@andreidinu please rebase

**Comment by jayvdb** [comment]
Merged in https://github.com/coala/coala/pull/4301

**Pull request closed without merging**